modules/smtp/modules.php

```diff
@@ -1133,7 +1133,7 @@ protected function output() {
                     '<textarea id="compose_body" name="compose_body" class="compose_body form-control" placeholder="'.$this->trans('Message').'">'.$this->html_safe($body).'</textarea>'.
                     (!$html ? '<label for="compose_body">'.$this->trans('Message').'</label>': '').
                 '</div>'.
-                '<div class="form-check mb-3"><input value="1" name="compose_delivery_receipt" id="compose_delivery_receipt" type="checkbox" class="form-check-input" /><label for="compose_delivery_receipt" class="form-check-label">'.$this->trans('Request a delivery receipt').'</label></div>';
+                '<div class="form-check mb-3"><input value="0" name="compose_delivery_receipt" id="compose_delivery_receipt" type="checkbox" class="form-check-input" /><label for="compose_delivery_receipt" class="form-check-label">'.$this->trans('Request a delivery receipt').'</label></div>';
```

Comment:
@amaninyumu1 This is the only one change needed here. Can you remove others before we merge ?

Reply:
I think because I modified the default values ​​of the checkbox  from 1 to 0

Reply:
I made some changes please review